### ISSUE TITLE
Feature Request: run ansible on different host or allow selecting a different path for ansible/ansible-playbook

### ISSUE THREAD

**Issue by msperl** [rundeck-plugins/ansible-plugin#327]
to support multiple ansible versions (and python) it would be nice if:
* either we could define a host on which to run the ansible command on instead of localhost
* and/or we could define a "custom" "ansible" and "ansible-playbook" which gets used (i.e: ansible2.9/ansible-playbook2.9) instead of the hardcoded ansible/ansible-playbook.

This would help migrating to newer versions of ansible/python and allow for delegations to dedicated nodes...

I understand that especially the first variant will be complicated from the perspective of gathering of node infos (because we would need to copy the ansible script to the new host and copy the results back to the host that runs rundeck).

Only workaround right now is to use the shell executor, but then we loose the means to select nodes in rundeck and run those nodes in parallel  or handle errors on a per host basis...

**Comment by jasenj1** [comment]
See #283

**Comment by fdevans** [comment]
Sorry we missed this. If it's still a problem please open a new Issue with current Rundeck version details, and a deeper description of how this can be repeated.
